atlasdb-feign/src/main/java/com/palantir/atlasdb/http/ExceptionCountingRefreshingClient.java

```diff
@@ -55,8 +55,7 @@ public static Client createRefreshingClient(Supplier<Client> baseClientSupplier)
 
     @Override
     public Response execute(Request request, Request.Options options) throws IOException {
-        try {
-            Response response = delegate().execute(request, options);
+        try (Response response = delegate().execute(request, options)) {
```

Comment:
haha this doesn't work of course, as we are returning a closed response in any case